examples/wordle/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Quick Start
 
-First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
+First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain.toml`](../../rust-toolchain.toml) file will be used by `cargo` to automatically install the correct version.
```

Comment:
Can we align the recommendations here with the other examples?
```suggestion
First, [install Rust] if you don't already have it.

Next, install the `cargo-risczero` tool and install the toolchain with:
```bash
cargo install risczero
cargo risczero install
```

Then, run the example with:
```bash
cargo run --release
```